File: web/src/types/notification.ts
/** 通知信息 */
export interface Notification {
  id: number;
  activityId: number;
  activityTitle: string;
  type: string;
  content: string;
  isRead: boolean;
  createTime: string;
}

/** 通知分页响应 */
export interface NotificationPageResponse {
  records: Notification[];
  total: number;
  pages: number;
  current: number;
}

File: src/main/java/com/campus/activity/entity/Notification.java
package com.campus.activity.entity;

import lombok.Data;
import java.time.LocalDateTime;

/**
 * 通知实体类
 * 用于存储用户收到的通知信息
 */
@Data
public class Notification {
    private Long id;
    private Long userId;
    private Long activityId;
    /** 通知标题（用于前端展示） */
    private String title;
    private String type;
    private String content;
    private Boolean isRead;
    private LocalDateTime createTime;
}
